core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -210,8 +210,7 @@ public long getTimeOut() {
 
   public TabletServerBatchWriter(ClientContext context, BatchWriterConfig config) {
     this.context = context;
-    this.executor = context.threadPools()
-        .createGeneralScheduledExecutorService(this.context.getConfiguration());
+    this.executor = context.getScheduledExecutor();
```

Comment:
This change makes me a bit uneasy, but maybe its ok.  Just thinking of the case batch writers for the metadata table and root table using this shared thread pool and root table failure not being able to process because a metadata table task is stuck.  Not sure if this can actually happen w/ this code though, if it does though it could casue problems for manager or tablets servers.

Reply:
Do you think I should undo this change in TabletServerBatchWriter?

Reply:
> Do you think I should undo this change in TabletServerBatchWriter?

Seems like that would be safer, do not know of a specific problem but its hard to conclude this task will always run when it needs to.  Because a shared pool at the context level processes task for all the data levels, seems like there can be dependencies that may cause problems and potentially prevent a metadata or root table update.

Reply:
Backed out those changes in a0507e1.